examples/minio_storage/README.md

```diff
@@ -0,0 +1,11 @@
+Example of MinIO for data and metadata store
+============================================
```

Comment:
It should give a small amount of background, you are fetching aluminum alloy metadata, for example